scripts/travis-deps.sh

```diff
@@ -3,6 +3,7 @@ set -x
 mkdir -p $HOME/gopath-staging
 cd $HOME/gopath-staging
 git clone --depth=1 --recursive --shallow-submodules https://github.com/storj/storj-vendor.git .
+./setup.sh
```

Comment:
did you mean for this to be here ?

Reply:
yep, doesn't work without that sadly.

Reply:
https://github.com/storj/storj-vendor/commit/4f6cac8fe9b258795a2ad808cf39d041ddec2e45